### PR TITLE
cleanup yarn scripts

### PR DESCRIPTION
**Pull request by jasalisbury**
The backend `test` and `lint` commands now only test/lint a single component. This should help clarify how to run tests or the linter
only the frontend or backend. Added readme for the frontend.

### CODE REVIEW

**Comment by kryswisnaskas** [comment]
Cool, the separation will be helpful.

I run the commands and  below are the results. 

Was also wondering if there is a shortcut for running the frontend tests alone.

Results:
`yarn docker:test` - works! 
`yarn docker:lint `- ESLint couldn't find the config "./node_modules/kcd-scripts/eslint.js" to extend from. Please check that the name of the config is correct.
`yarn docker:audit `- Error: Cannot find module '/app/scripts/audit.sh'

`yarn test` - works! Nice that it only tests the backend.
`yarn test:ci` - gives me 'JEST_JUNIT_OUTPUT_DIR' is not recognized as an internal or external command
`yarn lint `- $ ESLint couldn't find the config "developit" to extend from. Please check that the name of the config is correct. After adding eslint-config-developit (yarn add eslint-config-developit), I got another error - couldn't find the config "dev" to extend from

**Comment by jasalisbury** [comment]
You can run the frontend tests by going into the `frontend` dir and running `yarn test`

> `yarn docker:lint `- ESLint couldn't find the config "./node_modules/kcd-scripts/eslint.js" to extend from. Please check that the name of the config is correct.

This seems like a dependency issue, can you run `yarn docker:deps` and see if you still see the error?

> `yarn docker:audit `- Error: Cannot find module '/app/scripts/audit.sh'

I forgot I added the actual audit script in a different PR, I'll remove the audit command here and add it in the other PR.

> `yarn test:ci` - gives me 'JEST_JUNIT_OUTPUT_DIR' is not recognized as an internal or external command

I'm not sure how to handle this. On linux the yarn command sets some environment variables before running. Windows does not allow setting of environment variables in the same way. I can't find a cross platform way of setting the env variables needed.

> `yarn lint `- $ ESLint couldn't find the config "developit" to extend from. Please check that the name of the config is correct. After adding eslint-config-developit (yarn add eslint-config-developit), I got another error - couldn't find the config "dev" to extend from

Not sure what is going on here... do you have a `package-lock.json` file? If so can you delete it and reinstall dependencies?

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+| Yarn Command | Description |
+|-|-|
+| `yarn start` | Start the frontend development server |
+| `yarn build` | Build a production bundle |
+| `yarn test` | Start the test watcher |
+| `yarn test:ci` | Run unit tests a single time outputting results to xml files |
+| `yarn lint` | Run the linter |
+| `yarn lint:ci` | Run the linter outputting results to xml files |
+
+## Create React App
```
Comment (SarahJaine) [inline frontend/README.md:20]: Do we need all this (through the end of the page)? If it's helpful that's cool.  I'm wondering if we're giving free advertising to facebook. 😄

Comment (jasalisbury) [reply frontend/README.md:20]: :laughing: at the very least I think we should keep the link to create-react-app so if someone needs to mess with the build system they know where to look.

**Comment by SarahJaine** [comment]
@kryswisnaskas @jasalisbury Is this ready to merge? I want to rewrite the ci to use/store results from these updated commands. I can branch from this feature branch if need be.

**Pull request merged**